redbot/cogs/customcom/customcom.py

```diff
@@ -102,7 +102,10 @@ def get_now() -> str:
     async def create(self, ctx: commands.Context, command: str, *, response):
         """Create a custom command"""
         # Check if this command is already registered as a customcommand
-        if await self.db(ctx.guild).commands.get_raw(command, default=None):
+        if (
+            await self.db(ctx.guild).commands.get_raw(command, default=None)
+            or command in commands.RESERVED_COMMAND_NAMES
```

Comment:
The message send from this is "This command already exists. Use `[p]customcom edit` to edit it." rather than "There already exists a bot command with the same name.". Where the error is handled probably needs a check as well.

Reply:
Should be good to go now, moved the check in CC and added one which was missing.